examples/Recommendations.ipynb

```diff
@@ -0,0 +1,112 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demo Notebook for using the Reboot Recommendation Engine\n",
+    "\n",
+    "Run the cells in order, making sure to enter AWS credentials in the cell when prompted\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install git+https://github.com/RebootMotion/reboot-toolkit.git@v2.1.0#egg=reboot_toolkit > /dev/null\n",
+    "!wget https://raw.githubusercontent.com/RebootMotion/reboot-toolkit/main/examples/data/recommendation_metrics.csv\n",
+    "!echo \"Done Installing\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title Import Statements\n",
+    "\n",
+    "import reboot_toolkit as rtk\n",
+    "from reboot_toolkit import MovementType, setup_aws, MovementType, MocapType, Handedness\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title AWS Credentials\n",
+    "\n",
+    "boto3_session = setup_aws(org_id=\"org-mlbbiomech\", aws_default_region=\"us-west-1\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metrics_df = pd.read_csv(\"./recommendation_metrics.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = rtk.recommendation(boto3_session, metrics_df, MovementType.BASEBALL_PITCHING, MocapType.HAWKEYE, Handedness.RIGHT)"
```

Comment:
I'd probably want to add a bunch of comments here regarding what this does and how to use it, but I can add these comments in a further PR